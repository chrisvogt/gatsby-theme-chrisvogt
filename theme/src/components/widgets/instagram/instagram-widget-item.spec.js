import React from 'react'
import renderer from 'react-test-renderer'
import InstagramWidgetItem from './instagram-widget-item'

describe('InstagramWidgetItem', () => {
  const props = {
    post: {
      id: '0123456789',
      comments: { count: { commentsCount: 5 } },
      images: {
        standard_resolution: {
          height: 100,
          width: 100,
          url: 'https://cdn.chrisvogt.me/images/fake-instagram-image.jpg'
        }
      },
      likes: { count: { likesCount: 12 } },
      link: 'https://instagram.com/fake-image-link',
      type: 'IMAGE'
    }
  }

  it('matches the snapshot', () => {
    const tree = renderer.create(<InstagramWidgetItem {...props} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
