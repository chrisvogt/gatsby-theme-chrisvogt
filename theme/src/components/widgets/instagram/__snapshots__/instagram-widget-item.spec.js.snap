// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstagramWidgetItem matches the snapshot 1`] = `
<button
  className="css-sf4r4r-InstagramWidgetItem"
  onClick={[Function]}
  rel="noopener noreferrer"
>
  <img
    alt="Instagram post thumbnail"
    className="instagram-item-image css-ko57dg-InstagramWidgetItem"
    crossOrigin="anonymous"
    height="280"
    src="undefined?h=280&w=280&fit=crop&crop=faces,focalpoint&auto=format"
    width="280"
  />
</button>
`;
